src/components/Navigation/__stories__/Navbar.stories.tsx

```diff
@@ -4,6 +4,7 @@ import { ExecutionDetailsAppBarContent } from 'components/Executions/ExecutionDe
 import { mockExecution } from 'models/Execution/__mocks__/mockWorkflowExecutionsData';
 import { createMockExecution } from 'models/__mocks__/executionsData';
 import * as React from 'react';
+import { FeatureFlag, useFeatureFlagContext, FeatureFlagsProvider } from 'basics/FeatureFlags';
```

Comment:
nit: FeatureFlagsProvider is not needed here, and can be safely removed.

Reply:
fixed